src/transformers/models/yolos/configuration_yolos.py

```diff
@@ -74,8 +74,6 @@ class YolosConfig(PretrainedConfig):
             The number of detection tokens.
         use_mid_position_embeddings (`bool`, *optional*, defaults to `True`):
             Whether to use the mid-layer position encodings.
-        auxiliary_loss (`bool`, *optional*, defaults to `False`):
```

Comment:
Same comment here re param name

Reply:
@amyeroberts Hi, I have two question for you!
1. Do you want me to restore this param? or re param this name as 'use_auxiliary_loss'. I think it is the former one but just to make sure.
2. I think the better way is to revert whole commit especially in configuration_yolos.py. How about your opinion?

Reply:
1. Yes - let's go back to `use_auxillary_loss` 
2. However you'd like to do it :) All the commits are squashed before merging into main, so it doesn't matter too much. We should still include the auxiliary loss tests for these models though.